src/components/DisplayModal.tsx

```diff
@@ -21,18 +22,25 @@ export const DisplayModal: React.FC<DisplayModalProps> = ({
   setOpen,
   size,
   title,
+  popoverZIndex = zIndex.modal,
```

Comment:
> Note: I made the modal use a different z-index than zIndex.modal because the ChartToggle has a higher z-index than that, causing it to be visually on top of the modal (which is wrong).

I was playing around with the `ChartToggle` component and noticed that the `Container` doesn't really seem to do anything — when the page is loaded it has a height of 0, and removing it doesn't change anything (unless I am missing something...). Removing the container also means we can use the regular `zIndex.modal` here. Let me know if you find a bug with that though 😳

Reply:
you are completely correct thank you!!! ❤️